spec/unit/hanami/slice_configurable_spec.rb

```diff
@@ -28,15 +28,15 @@ def self.inherited(subclass)
         end
       end
     end
-
-    Hanami.app.prepare
   end
 
   context "subclass inside slice namespace" do
     before do
       module Main
         class MySubclass < TestApp::BaseClass; end
       end
+
+      Hanami.app.prepare
```

Comment:
I think we can actually remove all the `prepare` calls from the test altogether. I've just double checked this without them an everything passes. Would you mind taking care of that?

Reply:
👌 I realized that, but I thought it was there to reflect the actual setup. Removed!